helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -11,11 +11,11 @@ spec:
   groups:
   - name: app
     rules:
-    - alert: ManagementClusterAppFailedBatman
+    - alert: ManagementClusterAppFailedAtlas
```

Comment:
Diff is a pain but this just adds `ManagementClusterAppFailedLudacris` and orders the other rules.